src/main/resources/MANIFEST.MF

```diff
@@ -11,7 +11,7 @@ GitCommitIdDescribe: da9bc99
 GitBuildUserName: 
 GitRemoteOriginUrl: https://github.com/MyRobotLab/myrobotlab.git
 GitTags: 
-GitBranch: develop
+GitBranch: more_cli_fixes

```

Comment:
If this file is generated with the build, I would think it shouldn't be checked in?

Reply:
its a "basic" dev file - since locally running through the IDE you don't necessarily need nor want the generate file stuff to run .. there should be "something" checked in .. I tweak it - but rarely